src/main/java/firebase/sdk/dashboard/dao/SDKDao.java

```diff
@@ -38,56 +47,61 @@
    * @param releaseName The name of the release the SDKRelease object is enrolled 
    * in (ie M74, M75_hotfix, etc,).
    * @param libraryName The sdk's name.
-   * @return An SDKRelease object containing all the information about the SDKRelease from
+   * @return An SDKReleaseMetadata object containing all the information about the SDKRelease from
    * the data source.
    */
   public SDKReleaseMetadata getSDKReleaseMetadata(Platform platform, String releaseName, String libraryName);
 
-  //TODO: Define exception
   /**
-   * Adds the SDK from the data source.
+   * Adds the SDK to the data source.
    *
-   * @param platform The platform the sdk is being added to.
    * @param sdk The SDK being added.
    */
-  public void addSDK(Platform platform, SDK sdk);
+  public void addSDK(SDK sdk);
 
-  //TODO: Define exception
   /**
    * Removes the SDK from the data source.
    *
    * @param sdk The SDK being removed.
    */
   public void deleteSDK(SDK sdk);
 
-  //TODO: Define exception
   /**
-   * Creates a VersionMetadata object and adds it and the SDKReleaseMetada
-   * object to the data source.
+   * Adds the SDKReleaseMetadata object to the datasource.
+   *
+   * @param sdkRelease The SDKReleaseMetadata object that is being added.
+   */
+  public void addSDKRelease(SDKReleaseMetadata sdkRelease);
+
+  /**
+   * Deletes the SDKReleaseMetadata object from the datasource.
    *
-   * @param sdk The SDKReleaseMetadata object that is being added.
+   * @param sdkRelease The SDKReleaseMetadata object that is being deleted from the 
+   * data source.
    */
-  public void addSDKRelease(SDKReleaseMetadata sdk);
+  public void deleteSDKRelease(SDKReleaseMetadata sdkRelease);
 
-  //TODO: Define exception
   /**
-   * Deletes the SDKReleaseMetadata and its corresponding VerionMetadata 
-   * from the datasource.
+   * Adds the VersionMetadata object to the datasource.
    *
-   * @param sdk The SDKReleaseMetadata object that is being deleted from the 
+   * @param sdkVersion The VersionMetadata object that is being added.
+   */
+  public void addSDKVersion(VersionMetadata sdkVersion);
+
+  /**
+   * Deletes the VersionMetadata object from the datasource.
+   *
+   * @param sdkVersion The VersionMetadata object that is being deleted from the 
    * data source.
    */
-  public void deleteSDKRelease(SDKReleaseMetadata sdk);
+  public void deleteSDKVersion(VersionMetadata sdkVersion);
 
   /**
    * Updates the given SDKRelease object and registers the changes in the data 
    * source.
    *
-   * @param oldSDKRelease The SDKRelease before the changes were made.
-   * @param newSDKRelease The SDKRelease after the changes were made.
-   * @return Whether or not the SDKRelease was successfully modified.
+   * @param sdkRelease The SDKRelease after the changes were made.
```

Comment:
Expand on this - what changes and how does it translate in terms of SDKRelease object